docs/maui/essentials/badge.md

```diff
@@ -0,0 +1,127 @@
+    {
+        badge.SetCount(value);
+    }
+}
+```
+
+## Examples
+
+You can find an example of `Badge` in action in the [.NET MAUI Community Toolkit Sample Application](https://github.com/CommunityToolkit/Maui/blob/main/samples/CommunityToolkit.Maui.Sample/Pages/Essentials/BadgePage.xaml).
+
+## API
+
+You can find the source code for `Badge` over on the [.NET MAUI Community Toolkit GitHub repository](https://github.com/CommunityToolkit/Maui/blob/main/src/CommunityToolkit.Maui.Core/Essentials/Badge/IBadge.shared.cs).
+
+## Android
+
+In the world of Android, one size rarely fits all. This principle holds true when it comes to setting application badge counts, due to the lack of a standardized API provided by Android system for this functionality.
+
+Different Android launchers have chosen to implement badge counts in their unique way. Most launchers including popular ones like Nova Launcher, Microsoft Launcher, etc., have their specific methods to handle this feature.
+
+Consequently, to ensure that your application's badge notifications appear correctly across these diverse launchers, you must resort to programming specific code implementations for each launcher. This means adapting your code to the specific requirements of each launcher's unique badge count-indicating mechanism.
+
+Consider it as a hurdle that developers must overcome, in the path towards achieving a universal application experience. This is due to Android's flexible ecosystem that encourages diversity and customization.
+
+It's important to note that while it's feasible to cover the most popular Android launchers, it would be almost impossible to cater to every single one, given the sheer number of launchers available in the market.
+
+With the CommunityToolkit we provide you the way implement your own badge counter logic for providers you want to support. This is how you can do that:
+
+1. Implement `CommunityToolkit.Maui.ApplicationModel.IBadgeProvider` interface. See `SamsungBadgeProvider` implementation as example: [SamsungBadgeProvider](https://github.com/CommunityToolkit/Maui/blob/main/samples/CommunityToolkit.Maui.Sample/Platforms/Android/SamsungBadgeProvider.cs).
+1. In the Android `MainApplication` constructor map launcher identifier with your `BadgeProvider` implementation". For example for Samsung launcher it should look like this:
+```csharp
+public MainApplication(IntPtr handle, JniHandleOwnership ownership)
+		: base(handle, ownership)
+	{
+		var samsungProvider = new SamsungBadgeProvider();
+		BadgeFactory.SetBadgeProvider("com.sec.android.app.launcher", samsungProvider);
+		BadgeFactory.SetBadgeProvider("com.sec.android.app.twlauncher", samsungProvider);
+	}
+```
+1. Add required permissions to the `AndroidManifest.xaml`.  For example for Samsung launcher it should look like this:
+```xml
+<uses-permission android:name="com.sec.android.provider.badge.permission.READ" />
+<uses-permission android:name="com.sec.android.provider.badge.permission.WRITE" />
+```
```

Comment:
I wonder if we could/should add a little summary in at the end stating what the above example should achieve and how the reader can take the learnings further

e.g.

```suggestion
```

These changes are enough for your application to run on an Android based Samsung device and correctly update the app icons badge number. It also gives you a starting point and guide as to how you could start implementing support for other launchers within your application.
```

**I don't think the suggestion will apply correctly as it is around backticks and marking the wrong ones as the close of the suggestion.**